azurelinuxagent/common/utils/networkutil.py

```diff
@@ -119,7 +119,35 @@ class AddFirewallRules(object):
     Please make sure to not log anything in any function this class.
     """
 
-    __ACCEPT_COMMAND = "-A"
+    # -A adds the rule to the end of the iptable chain
+    __APPEND_COMMAND = "-A"
+
+    # -I inserts the rule at the index specified. If no number specified the rules get added to the top of the chain
+    # iptables -t security -I OUTPUT 1 -d 168.63.129.16 -p tcp --destination-port 53 -j ACCEPT -w and
+    # iptables -t security -I OUTPUT -d 168.63.129.16 -p tcp --destination-port 53 -j ACCEPT -w both adds the rule as the first rule of the chain
+    __INSERT_COMMAND = "-I"
+
+    # -D deletes the specific rule in the iptable chain
+    __DELETE_COMMAND = "-D"
+
+    # -C checks if a specific rule exists
+    __CHECK_COMMAND = "-C"
+
+    @staticmethod
+    def get_append_command():
```

Comment:
these getters seem an overkill (probably i am missing something) why not use the constants you defined above directly?

also, they don't seem to be used outside this class, so they should be private

Reply:
Since we are using the insert , check and delete command in update ,test_update and default files I used getters .So I thought of using it everywhere for consistency.  Any other suggestions to avoid them ?

Reply:
We can just make these variables as public and use them directly instead of using a getter :)